examples/django/django_exampleapp/weather_app/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.templatetags.static import static
+from django.urls import path
+from rest_framework.authtoken.views import obtain_auth_token
+
+from . import views
+
+urlpatterns = [
+    path('', views.hello),
+    path('date/', views.getdate),
+    path('locations/', views.get_location_details),
```

Comment:
I think adding these trailing slashes means that they're required in the curl which is _probably_ not what we want.

Example:
1. Run: `pip3 install -r requirements.txt`
2. Run: `python manage.py runserver 3000`
3. Run: `curl -i "http://localhost:3000/locations"` Get:
```
HTTP/1.1 301 Moved Permanently
Date: Wed, 05 Jan 2022 23:33:31 GMT
Server: WSGIServer/0.2 CPython/3.9.7
Content-Type: text/html; charset=utf-8
Location: /locations/
Content-Length: 0
X-Content-Type-Options: nosniff
Referrer-Policy: same-origin
```
4. Run: `curl -i "http://localhost:3000/locations/"` (note that trailing `/` character) Get:
```
HTTP/1.1 200 OK
Date: Wed, 05 Jan 2022 23:34:46 GMT
Server: WSGIServer/0.2 CPython/3.9.7
Content-Type: application/json
X-Frame-Options: DENY
Content-Length: 34
X-Content-Type-Options: nosniff
Referrer-Policy: same-origin

["pune", "austin", "santabarbara"]%
```

Reply:
An alternative would be to use `curl -iL ...` where the `-L` will allow the `Location` header from the 301 response. However, I think you're right Thom that we should look at removing the trailing slash.

Reply:
I have updated the trailing slashes in the README examples as it is the convention followed for the routes in URL patterns in django, so removing the trailing slash would not work for our application.